docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -50,6 +50,9 @@ NOTE: the `list()` method might be surprising at first. It takes fragments of Pa
 That makes for very concise but yet readable code.
 MongoDB native queries are also supported.
 
+NOTE: what you just see is the link:https://www.martinfowler.com/eaaCatalog/activeRecord.html[active record pattern], sometimes just called the entity pattern.
```

Comment:
Please just copy the changes I proposed in the other doc to this one :)

Reply:
OK, will do it